ui_testing/pages/order_page.py

```diff
@@ -258,6 +258,84 @@ def duplicate_suborder(self):
                                                                        destination_element='order:duplicate_table_view')
         duplicate_element.click()
 
+    # this method to be used while you are order's table with add page ONLY, and you can get the required data by sending the index, and the needed fields of the suborder
+    def get_suborder_data(self, sub_order_index=0, departments=True, material_type=True, articles=True, test_plan=True, test_unit=True, test_date=False, shipment_date=False):
+        table_suborders = self.base_selenium.get_table_rows(element='order:suborder_table')
+        required_suborder = self.base_selenium.get_row_cells_id_elements_related_to_header(row=table_suborders[sub_order_index],
+                                                                                             table_element='order:suborder_table')
+        response = {
+            "departments": "",
+            "material_types": "",
+            "article": "",
+            "test_plan": "",
+            "test_unit": "",
+            "test_date": "",
+            "shipment_date": ""
+        }
+        if departments :
+            response["departments"]="|".join(list(map(lambda s: str(s)[1:], required_suborder["departments"].text.split("\n")) ))
+        if test_plan :
+            response["test_plan"]="|".join(list(map(lambda s: str(s)[1:], required_suborder["testPlans"].text.split("\n")) ))
+        if test_unit :
+            response["test_unit"]="|".join(list(map(lambda s: str(s).split(' No:')[0][1:], required_suborder["testUnits"].text.split("\n")) ))
+        if articles :
+            response["article"]=required_suborder["article"].text.split("\n")[0]
+        if material_type :
+            response["material_types"]=required_suborder["materialType"].text.split("\n")[0]
+        if shipment_date :
+            pass
+        if test_date :
+            pass
+        return response
+
+    def remove_testplan_by_name(self, index, testplan_name):
```

Comment:
this method gonna remove by choosing the already selected item, so it will remove it from the selected items because that's the behavior of the ng-select, the selector we are using